internal/netutil/ident.go

```diff
@@ -41,7 +41,7 @@ var (
 )
 
 // ConnUserid returns the uid that owns the given localhost connection.
-// The returned error is ErrNotFound if the connection wasn't found.
+// If the connection is not found, returns -1 and ErrNotFound.
```

Comment:
This invites the question: for other types of errors, might it not return -1? In that case, the caller can't depend on this being safe anyway:

     uid, _ := netutil.ConnUserid(c)

In which case there's no point in the -1 guardrail for those ignoring errors anyway.

So we should either go all the way to Go-idiomatic (zero-value, non-nil-error) or if we really want the -1 guardrail, go all the way and say that on any error, it's -1. But that feels so weird to promise.

I think we should just abandon all negative one and be idiomatic and return zero values on errors.